### PR TITLE
refs T32414: add csrf token to registration

### PR DESCRIPTION
**Pull request by mussbach**
**Ticket:** https://yaits.demos-deutschland.de/T32414

This PR adds a csrf token to citizen and organization registration to avoid forms being misused for spam

### How to review/test
register an organization and replay post request via browser or curl. With this patch no new organization should be added 

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

**Review by hwiem — COMMENTED** [review]
Just 2 nitpicks, rest of FE lgtm

**Review by AlexMenschoid — DISMISSED** [review]
BE lgtm

File: client/js/components/user/citizenRegisterForm/CitizenRegisterForm.vue [hunk 1]
```diff
@@ -126,6 +131,12 @@ export default {
       type: String,
       required: true,
       default: ''
+    },
+
+    csrfToken: {
+      type: String,
+      required: true,
+      default: ''
```
[the remark this answers is not in the document]
Comment (mussbach) [reply client/js/components/user/citizenRegisterForm/CitizenRegisterForm.vue]: indeed, thank you df6105c4

File: client/js/components/user/orgaRegisterForm/OrgaRegisterForm.vue [hunk 2]
```diff
@@ -186,6 +193,11 @@ export default {
     customer: {
       type: String,
       required: true
+    },
+    csrfToken: {
+      type: String,
+      required: true,
+      default: ''
```
[the remark this answers is not in the document]
Comment (mussbach) [reply client/js/components/user/orgaRegisterForm/OrgaRegisterForm.vue]: df6105c4

**Pull request merged**